### ISSUE TITLE
Loading from snapshot loss is much higher than expected

### ISSUE THREAD

**Issue by tarunsharma1** [DeepLabCut/DeepLabCut#474]
Using DLC 2.0.8 on Ubuntu 18

I have seen this in multiple issues but it has not been addressed. When I change the init_weights in pose_cfg.yaml to a snapshot at which the training was previously interrupted, the loss is really high (much higher than expected if continuing from that snapshot). It is equivalent to that when the network was started from imagenet weights. I am not sure if the snapshot weights are being loaded correctly.

**Comment by MMathisLab** [comment]
It is loading the backbone network pretrained weights correctly, but previously we did not load the trained deconvolutional layers. Now in 2.1+ that has changed. If you are concerned, please update, as this is loaded by default now.
```
deeplabcut.train_network?

Signature: deeplabcut.train_network(config, shuffle=1, trainingsetindex=0, max_snapshots_to_keep=5, displayiters=None, saveiters=None, maxiters=None, allow_growth=False, gputouse=None, autotune=False, keepdeconvweights=True)
...

keepdeconvweights: bool, default: true
Also restores the weights of the deconvolution layers (and the backbone) when training from a snapshot. Note that if you change the number of bodyparts, you need to set this to false for re-training.


```

**Comment by tarunsharma1** [comment]
ohhh ..thank you. Is there any other way I can load the trained deconv weights? As I have already made changes to the code and updating it would erase all those changes.

**Comment by MMathisLab** [comment]
Perhaps you have have another environment that has the latest version, or of course check out the edits we made to the code. If you’re updates are generally useful, also feel free to make a PR!

**Comment by tarunsharma1** [comment]
Fixed it by replacing variables_to_restore = slim.get_variables_to_restore(include=["resnet_v1"])
to variables_to_restore = slim.get_variables_to_restore()
